### PR TITLE
added main for jspm compat

### PR DESCRIPTION
**Pull request by digital-synapse**
https://github.com/google/lovefield/issues/80

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "lovefield",
   "version": "2.0.62",
+  "main": "dist/lovefield.js",
```
Comment (arthurhsu) [inline package.json]: Should use the minified version as main: dist/lovefield.min.js.

Comment (digital-synapse) [inline package.json]: Usually main points to the development version. Example. https://github.com/jashkenas/underscore/blob/master/package.json

**Comment by arthurhsu** [comment]
LGTM. However, I need you to sign the contributor license agreement form as instructed in https://github.com/google/lovefield/blob/master/CONTRIBUTING.md before I can merge your pull request. Please let me know when you have it signed. Thanks!

**Comment by digital-synapse** [comment]
ok you should have it.

**Comment by arthurhsu** [comment]
Merged c315be3969bc905313b702a8fb3525e1ee8b74e1

**Pull request closed without merging**